src/commands/deploy/deploy.ts

```diff
@@ -177,12 +175,17 @@ async function deployToSasViya(
       ),
       log
     )
-    console.log(
-      `Job execution completed! Log is available at ${path.join(
+    process.logger?.success(
+      `Deployment completed! Log is available at ${path.join(
         logFilePath,
         `${path.basename(deployScript).replace('.sas', '')}.log`
       )}`
     )
+
+    if (!!target.streamConfig?.streamWeb) {
+      const webAppStreamUrl = `${target.serverUrl}/SASJobExecution?_PROGRAM=${target.appLoc}/services/${target.streamConfig.streamServiceName}`
```

Comment:
This won't work for SAS 9 - I think we have some logic for this in the adapter (SASJobExecution vs SASStoredProcess)

Reply:
This line is in the `deployToSasViya` function.
But that's a valid point regardless - I've now added a similar log for SAS9 with the correct URL.